### PR TITLE
Readme proposal

### PR DESCRIPTION
**Pull request by h-mayorquin**
This is the structure of the readme that, in my understanding, we agreed upon on the team meeting today. If you could take a look at how the Readme looks like it would be great.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,20 @@ You can install the latest release of the package with pip:
...
+* [Convert all courtship sessions, Cowley 2022](https://github.com/catalystneuro/murthy-lab-to-nwb/blob/main/src/murthy_lab_to_nwb/cowley2022mapping/cowley2022mapping_courtship_convert_all.py)
+* [Convert all imaging sessions, Cowley 200](https://github.com/catalystneuro/murthy-lab-to-nwb/blob/main/src/murthy_lab_to_nwb/cowley2022mapping/cowley2022mapping_imaging_convert_all.py)
+* [Convert one session, Li 2022](https://github.com/catalystneuro/murthy-lab-to-nwb/blob/main/src/murthy_lab_to_nwb/li2022ecephys/li2022ecephys_convert_session.py)
+
+## Installation from Github
+Another option is to install the package directly from Github. This option has the advantage that the source code can be modify if you need to amend or modify some of the code provided by us. To install the conversion from GitHub you will need to use `git` ([installation instructions](https://github.com/git-guides/install-git)). We also recommend the installation of `conda` ([installation instructions](https://docs.conda.io/en/latest/miniconda.html)) as it contains all the required machinery in a single and simple instal
```
Comment (CodyCBakerPhD) [inline README.md]:
```suggestion
Another option is to install the package directly from Github. This option has the advantage that the source code can be modifed if you need to amend some of the code we originally provided to adapt to future experimental differences. To install the conversion from GitHub you will need to use `git` ([installation instructions](https://github.com/git-guides/install-git)). We also recommend the installation of `conda` ([installation instructions](https://docs.conda.io/en/latest/miniconda.html)) as it contains all the required machinery in a single and simple instal
```

File: README.md [hunk 2]
```diff
@@ -19,9 +30,11 @@ cd murthy-lab-to-nwb
 conda env create --file make_env.yml
 conda activate murthy-lab-to-nwb-env
 ```
-This create a [conda environment](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/environments.html) which isolates the conversion from your system. We recommend that you run all your conversion related tasks and analysis from that environment to minimize the intereference of this code with your own system.
+
+This creates a [conda environment](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/environments.html) which isolates the conversion code from your system libraries.  We recommend that you run all your conversion related tasks and analysis from the created environment in order to minimize configuration or libraries issues.
```
Comment (CodyCBakerPhD) [inline README.md]:
```suggestion
This creates a [conda environment](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/environments.html) which isolates the conversion code from your system libraries.  We recommend that you run all your conversion related tasks and analysis from the created environment in order to minimize issues related to package dependencies.
```

**Comment by CodyCBakerPhD** [comment]
Couple of word change suggestions, other than that looks great to me!

**Comment by h-mayorquin** [comment]
Thanks @CodyCBakerPhD

**Pull request merged**